client/onboarding/steps/business-details.tsx

```diff
@@ -84,33 +84,13 @@ const BusinessDetails: React.FC = () => {
 
 	return (
 		<>
-			{ selectedCountry && (
```

Comment:
a bit above this file:
`'InlineNotice' is defined but never used`
could you remove it please?